[Feature] `randomInt` and `randomItem` utils #202 (#398)
studiometa/js-toolkit: packages/docs/utils/randomInt.md

# randomInt

Get a random integer between bounds.

## Usage

```js
import { randomInt } from '@studiometa/js-toolkit/utils';

> randomInt(10);
> randomInt(20, 10);

Reviewer: **suggestion:** add possible output as comment

```suggestion
randomInt(10); // an integer between 0 and 10 
randomInt(20, 10); // an integer between 10 and 20
```

```

### Parameters

> - `max` (`number`): maximum value

Reviewer: **chore:** this should be `a` 

```suggestion
- `a` (`number`): lower bound value
```

> - `min` (`number`): minimum value (default: `0`)

Reviewer: **chore:** this should be `b`

```suggestion
- `b` (`number`): upper bound value (default: `0`)
```

### Return value

> This function returns a random `number` between the `min` and `max` values.

Reviewer: **chore:** replace `min` and `max` by `a` and `b`

```suggestion
This function returns a random `number` between the `a` and `b` values.
```
